### ISSUE TITLE
Postprandial exacerbation

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10573]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 32644615 Symptoms of autonomic dysfunction in nutcracker syndrome are rare but include hypotension, syncope, and tachycardia. Cases of nausea and weight loss, unexplained pelvic pain, and left flank pain with postprandial exacerbation have been described. 

**Parent term (use hpo.jax.org/app)**
Aggravated by HP:0025285

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Renal nutcracker syndrome ORPHA:71273

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 

Adding new term: 
- ID: HP:6000932
- label: Postprandial exacerbation
- parents: [Aggravated by (HP:0025285)](https://hpo.jax.org/app/browse/term/HP:0025285)
- pmid: 
- def: Applied to a sign or symptom that is more severe following a meal.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10573
- synonyms:
